docs/keystore.md

```diff
@@ -1 +1,117 @@
 # Keystore management
+
+## Key types
+
+There are three types of key representations that are used:
+
+- `cyber`
+  - Derived from account keys generated by `cyberdcli keys add`
+  - Used to receive funds
+  - e.g. `cyber15h6vd5f0wqps26zjlwrc6chah08ryu4hzzdwhc`
+
+- `cybervaloper`
+  - Used to associate a validator to its operator
+  - Used to invoke staking commands
+  - e.g. `cybervaloper1carzvgq3e6y3z5kz5y6gxp3wpy3qdrv928vyah`
+
+- `cyberpub`
+  - Derived from account keys generated by `cyberdcli keys add`
+  - e.g. `cyberpub1zcjduc3q7fu03jnlu2xpl75s2nkt7krm6grh4cc5aqth73v0zwmea25wj2hsqhlqzm`
+
+- `cybervalconspub`
+  - Generated when the node is created
+  - Get this value with `cyberdcli tendermint show-validator`
+  - e.g. `cybervalconspub1zcjduepq0ms2738680y72v44tfyqm3c9ppduku8fs6sr73fx7m666sjztznqzp2emf`
+
+## Generate keys
+
+You'll need an account private and public key pair \(a.k.a. `sk, pk` respectively\) to be able to receive funds, send txs, bond tx, etc.
+
+To generate a new _secp256k1_ key:
+
+```bash
+cyberdcli keys add <account_name>
+```
+
+Next, you will have to create a passphrase to protect the key on disk. The output of the above
+command will contain a _seed phrase_. It is recommended to save the _seed phrase_ in a safe
+place so that in case you forget the password, you could eventually regenerate the key from
+the seed phrase with the following command:
+
+```bash
+cyberdcli keys add --recover
+```
+
+Also, you can import your ETH account to `cyberdcli`:
+
+```bash
+cyberdcli keys add import_private <account_name>
+```
+by ETH private key
+
+If you check your private keys, you'll now see `<account_name>`:
+
+```bash
+cyberdcli keys show <account_name>
+```
+
+View the validator operator's address via:
+
+```shell
+cyberdcli keys show <account_name> --bech=val
+```
+
+You can see all your available keys by typing:
+
+```bash
+cyberdcli keys list
+```
+
+View the validator pubkey for your node by typing:
+
+```bash
+cyberdcli tendermint show-validator
+```
+
+Note that this is the Tendermint signing key, _not_ the operator key you will use in delegation transactions.
+
+::: danger Warning
+We strongly recommend _NOT_ using the same passphrase for multiple keys. The cyberd team and the cyber•Congress team will not be responsible for the loss of funds.
+:::
+
+## Keystore
+
+All user keypairs stored locally at `PATH_TO_CYBERD/cli/keys` folder. You can backup this folder for easy migrating between testnets.
+
+The node keypair stored in `priv_validator_key.json` at `PATH_TO_CYBERD/daemon/config` folder. You can delete it and restart docker if you want to change this keypair. The new pair will create automatically. Or you can backup it too and insert in the same folder at new testnet if you want same validator pubkey.
```

Comment:
Need to describe node and validators keys separetly

Reply:
@litvintech 
You mean two separate docs?

Reply:
@SaveTheAles you mentioned only validator key, there is also node key

Reply:
@litvintech I'm not sure that understand you correct. Have made some fixes. Can you take a look now?